pulpcore/pulpcore/app/serializers/base.py

```diff
@@ -76,6 +76,16 @@ def view_name_for_model(model_obj, view_action):
     raise LookupError('view not found')
 
 
+def validate_unknown_fields(initial_data, defined_fields):
+    """
+    This will raise a `ValidationError` if a serializer is passed fields that are unknown.
+    """
+    unknown_fields = set(initial_data.keys()) - set(defined_fields.keys())
```

Comment:
Prefer: set(initial_data) and set(defined_fields)

Reply:
Good catch, thanks.